### DISCUSSION TITLE
코드블럭 borderline 이슈

### DISCUSSION THREAD

**Discussion by hakyung-park** [pronist/hELLO#86] [Ideas]
안녕하세요! 

어제 처음 티스토리를 접했는데, 너무 세련된 스킨을 배포해주셔서 감사하게 잘 사용하고 있습니다 :) 
발견한 트러블 1 개와 건의사항 2 건을 말씀드리고 갑니다. 

1. 저장해놓은 서식을 불러와서 작성하는 경우, 서식에 있는 코드블럭을 재사용하게 되면 다음 사진처럼 하얀색 굵은 테두리가 생깁니다. 애초에 바깥선 색상이 흰색으로 설정되어 있는 것 같습니다. 모서리에 흰 색 점으로 보이네요!
![trouble1](https://user-images.githubusercontent.com/79737936/111234964-0decaa00-8633-11eb-94a1-ed6357ea69de.png)

2. 건의사항 (1)
코드블럭에 사용되는 하이라이팅을 설정하게 해주신 것 처럼 폰트 설정도 차후에 조금 추가되었으면 좋겠습니다. 
혹시 추가 예정이면 고려해주시면 감사드리겠습니다.
구체적으로 원하는 폰트는 DejaVuSansMono 이고 기능은 사이즈 조절입니다. 

3. 건의사항 (2)
제가 티스토리를 아직 다 파악한게 아니라서 스킨을 제작하면서 가능한 것인지 모르겠습니다.
서식을 불러와서 작성하는 경우, 1) 제목이 유지되고, 2) 해시태그도 저장할 수 있게 되었으면 하고 바랍니다.

다시 한 번 감사합니다 :)

**Comment by pronist** [comment]
1번 문제점의 경우 살펴봐야 할듯 하네요.

건의사항 (1)은 반영될 가능성이 희박합니다.
건의사항 (2)는 제작부분에서 처리할 수 있는 것이 아니라, 티스토리 개발팀에 문의주셔야 합니다 :)

**Comment by hakyung-park** [comment]
답변 감사합니다! 폰트는 스스로 해결했습니다 :)

**Comment by pronist** [comment]
1번 문제는 일반 배경을 날리는것으로 처리되었습니다.
